AdminUi/src/AdminUi/Controllers/IdentitiesController.cs

```diff
@@ -50,6 +54,41 @@ public async Task<CreatedResult> CreateIndividualQuota([FromRoute] string identi
         var createdIndividualQuotaDTO = await _mediator.Send(new CreateQuotaForIdentityCommand(identityAddress, request.MetricKey, request.Max, request.Period), cancellationToken);
         return Created(createdIndividualQuotaDTO);
     }
+
+    [HttpGet("{address}")]
+    [ProducesResponseType(typeof(GetIdentityByAddressResponse), StatusCodes.Status200OK)]
+    [ProducesError(StatusCodes.Status404NotFound)]
+    public async Task<IActionResult> GetIdentityByAddressAsync([FromRoute] string address, CancellationToken cancellationToken)
+    {
+        var identity = await _mediator.Send(new GetIdentityByAddressQuery(address), cancellationToken);
+        var identityWithQuotas = await _mediator.Send(new GetIdentityQuotasByAddressQuery(address), cancellationToken);
```

Comment:
Instead of the `GetIdentityQuotasByAddressQuery`, there should be a `GetIdentityQuery` in the Quotas module as well, which for now has two properties: `Address` and `Quotas`.